### DISCUSSION TITLE
Error page with react-router-dom, vite and react app

### DISCUSSION THREAD

**Discussion by AhmedTakeshy** [vercel/community#2177] [Help]
### Summary

I'm using the last version of react-router-dom and it has an error element feature to use as a component for error pages, but after I deployed my app. When I browse to any segment that does not exist in my app and I should get my custom error page but I get the default 404 page from Vercel instead.

### Additional information

Here is the live link for the app: https://wanderly.vercel.app/

### Example

https://github.com/AhmedTakeshy/Wanderly

**Comment by sundaram2021** [comment]
The Vercel routing configuration may not be set up correctly to handle all the routes in your application. In this case, you should check the Vercel configuration files (e.g., vercel.json) and make sure that the routes property is set up correctly to handle all the paths in your application.

The Vercel deployment may not be picking up the latest changes to your codebase. In this case, you can try redeploying your application to Vercel to see if that resolves the issue.

There may be an issue with the client-side routing configuration in your application. When using client-side routing with libraries like react-router-dom, the server needs to be configured to always serve the root index.html file, regardless of the URL path. This is because the client-side router will take over and handle the routing based on the URL path.


Try using any of these
